src/approaches/oracle_approach.py

```diff
@@ -7,17 +7,24 @@
 """
 
 from typing import List, Sequence, Set
+import itertools
 import numpy as np
 from predicators.src.approaches import TAMPApproach
 from predicators.src.envs import create_env, BlocksEnv, PaintingEnv, PlayroomEnv
 from predicators.src.structs import NSRT, Predicate, State, \
     ParameterizedOption, Variable, Type, LiftedAtom, Object, Array
 from predicators.src.settings import CFG
+from predicators.src.envs.behavior_options import (
+    navigate_to_param_sampler,
+    grasp_obj_param_sampler,
+    place_ontop_obj_pos_sampler,
+)
+from predicators.src.envs import get_env_instance
 
 
 class OracleApproach(TAMPApproach):
-    """A TAMP approach that uses hand-specified NSRTs.
-    """
+    """A TAMP approach that uses hand-specified NSRTs."""
```

Comment:
we've (tried to) used the convention that triple quotes are always on the second line -- if you feel strongly that it should be this way, let us know and we can change it everywhere in a separate PR. this comment applies elsewhere in this PR too

Reply:
I see - will fix!